README.md

```diff
@@ -1 +1,58 @@
+
+Beenfits for the user:
+- A single location for add-ons to Silk.NET - no need to go hunting!
+- Readily-available access to mature projects that can be trusted and are endorsed by the Silk.NET team
+
+Benefits to the community project maintainers:
+- More exposure to more users
+- Greater user buy-in given the "Trusted by the Silk.NET Team" image
+
+Benefits for the Silk.NET maintainers:
+- No need to envelop these projects into the main library, as a result:
+    - Less work to maintain the Silk.NET library
+    - Less risk of scope creep
+
+## Get your project involved!
+
+Simply open an issue using the "Project Submission" template and the Silk.NET team will review your project as soon as possible!
+
+### Points to Note
+
+- You keep all maintenance responsibility
+- You keep admin permissions on the GitHub Repository.
+- You keep full copyright of the project, and do not need to grant anyone any additional rights (such as trademark or patent licenses), though the permissive license requirement 
+- You can withdraw from the Silk.NET Community at any time (contact us on Discord)
+- You still have full control over library design, code, and maintenance decisions & policies, except where otherwise noted in the "Onboarding Requirements"
+- You may request a channel in the Silk.NET Discord Server for development discussion
+
+### Prerequisites 
+
+1. You use Silk.NET predominantly for any low-level interoperation with libraries that Silk.NET provide bindings for, and help promote the Silk.NET as the one bindings library you need for low-level graphics.
+2. You are stable enough to be recommended to our community, and the project is in a good enough state that our userbase can start integrating your work into theirs.
+    - Betas and pre-releases are okay, so long as the primary advertised functionality works (i.e. it "does what it says on the tin") 
+3. You solve a problem which the Silk.NET maintainers team recognises as a problem, and they believe that your project is useful when building libraries and applications that use Silk.NET.
+4. Your leadership skills and/or familiarity with the Silk.NET toolset is recognised by the Silk.NET maintainers (e.g. you've proven to be a friendly face in Discord, you're a known contributor on GitHub, and/or are a known Silk.NET evangelist) 
+5. You use a permissive copyright license that is suitable for unrestricted out-of-the-box consumption
+     - MIT or MIT-compatible licenses (e.g. Apache 2.0, BSD-3, etc) are recommended
+     - The license must permit dynamic and static linking without conditions that affect the licensing of the consuming project
+
+### Onboarding Requirements
+
+- You will need to adopt the Contributor Covenant Code of Conduct, and ensure your community doesn't break the rules. A copy of this is provided in the [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md) file. 
+- You will need to use Discord and join the Silk.NET Discord Server so that we can maintain contact with you.
```

Comment:
Maybe we should also ask for an email, to everyone uses discord all the time and in case things burn / someone goes missing?

Reply:
```suggestion
- You will need to use Discord and join the Silk.NET Discord Server so that we can maintain contact with you, ideally providing an alternative contact method (such as an email address) in case we can't.
```